language/de/migrator.php

```diff
@@ -64,9 +65,12 @@
 
 
 	'MODULE_ERROR'						=> 'Bei der Erstellung eines Moduls ist ein Fehler aufgetreten: %s',
+	'MODULE_EXISTS'						=> 'Ein Modul existiert bereits: %s',
+	'MODULE_EXIST_MULTIPLE'				=> 'Es existieren mehrere Module mit dem angegebenen übergeordneten Sprachnamen: %s. Verwende die Nach-Oben-/Nach-Unten-Schaltflächen, um die Position des Moduls festzulegen.',
```

Comment:
In welchem Bezug wird das benutzt? für "DE" wenn "de" und "de-x-sie" zur Verfügung stehen (oder "en" und "en-us" z.B.)?